Libraries/Boards/MAX78002/EvKit_V1/board.mk

```diff
@@ -35,13 +35,25 @@ ifeq "$(BOARD_DIR)" ""
 $(error BOARD_DIR must be set)
 endif
 
+TFT?=ADAFRUIT
```

Comment:
Used `?=` so it can be overridden from project.mk

Reply:
Agree